### ISSUE TITLE
`go get github.com/kyokomi/xcrawl` failed

### ISSUE THREAD

**Issue by hkurokawa** [kyokomi/xcrawl#1]
I tried to `go get` this repository to have an error like below.

```shell
% go get github.com/kyokomi/xcrawl
# cd .; git clone https://github.com/kyokomi/amber_lambda /Users/hiroshi/go/src/github.com/kyokomi/amber_lambda
Cloning into '/Users/hiroshi/go/src/github.com/kyokomi/amber_lambda'...
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
package github.com/kyokomi/amber_lambda/crawl: exit status 128
```

I guess the library name was changed from `amber_lambda` to `crawl`. Could you check this? Thanks.

**Comment by kyokomi** [comment]
Thanks for the report. I fixed import here #2 :bow

**Comment by hkurokawa** [comment]
Appreciate your prompt response. I'm closing the issue. :tada:
